lighthouse/blueprints/cherrypicked_plates.py

```diff
@@ -28,9 +29,16 @@
 def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
 
     try:
+        user_id = request.args.get("user_id", "UNKNOWN")
```

Comment:
In the `plate-events/create?...` endpoint I require that the user_id is specified. Should we align the behaviour for consistency? Either the plate event creation methods should default to an UNKNOWN user, or this endpoint should require a user_id

Reply:
For the uat version I added it optional in case was not being sent now. it seems that is always sent so we can use same behaviour as with events.